### ISSUE TITLE
[BUG]: Outdated Jar file & error that Spark version 2.4.1 is unsupported

### ISSUE THREAD

**Issue by bamurtaugh** [dotnet/spark#282]
**Describe the bug**
This may be a bug, or I may have missed a step somewhere/be trying to run an invalid command.

I'm following the instructions [here](https://github.com/bamurtaugh/spark/blob/master/docs/building/windows-instructions.md#using-net-core-cli-for-net-core-21x) to build and run the example apps on Windows.

**To Reproduce**
I'm at the stage of `spark-submit`. Specifically, I'm trying to run Microsoft.Spark.Examples.Sql.Basic.

I navigate to the publish folder, then run the submit command:

`cd C:\Users\brmurtau\Documents\GitHub\sparkedits\artifacts\bin\Microsoft.Spark.CSharp.Examples\Debug\netcoreapp2.1\win10-x64\publish`

`spark-submit --class org.apache.spark.deploy.dotnet.DotnetRunner --master local C:\Users\brmurtau\Documents\GitHub\sparkedits\src\scala\microsoft-spark-2.3.x\target\microsoft-spark-2.3.x-0.5.0.jar dotnet %SPARK_HOME%\examples\src\main\resources\people.json`

However, although Spark version 2.4.1 is supported by .NET for Apache Spark v0.5.0, I get the error:

```
Exception in thread "main" java.lang.IllegalArgumentException: Unsupported spark version used: 2.4.1. Normalized spark version used: 2.4.1. Supported versions: 2.3.3, 2.3.4, 2.3.0, 2.3.1, 2.3.2
        at org.apache.spark.deploy.dotnet.DotnetRunner$.validateSparkVersions(DotnetRunner.scala:167)
        at org.apache.spark.deploy.dotnet.DotnetRunner$.main(DotnetRunner.scala:46)
        at org.apache.spark.deploy.dotnet.DotnetRunner.main(DotnetRunner.scala)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at org.apache.spark.deploy.JavaMainApplication.start(SparkApplication.scala:52)
        at org.apache.spark.deploy.SparkSubmit.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:849)
        at org.apache.spark.deploy.SparkSubmit.doRunMain$1(SparkSubmit.scala:167)
        at org.apache.spark.deploy.SparkSubmit.submit(SparkSubmit.scala:195)
        at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:86)
        at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:924)
        at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:933)
        at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
```

Also, on the page of directions (I linked to above), it says to use `C:\github\dotnet-spark\src\scala\microsoft-spark-2.3.x\target\microsoft-spark-2.3.x-1.0.0-alpha.jar`, but I believe this jar is outdated and should be updated to `0.5.0`?

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Spark 2.4.1, .NET for Apache Spark 0.5.0 

My `SPARK_HOME` variable is set to `C:\bin\spark-2.4.1-bin-hadoop2.7\`. I've updated `DOTNET_WORKER_DIR` to `C:\bin\Microsoft.Spark.Worker-0.5.0`.

**Additional context**
In summary, this is a 2-part issue on the same set of directions (and I'm not sure if they're combining to impact my ability to run spark-submit):
1) The directions say to use an alpha jar file, which I believe is outdated.
2) I'm getting an error that 2.4.1 is unsupported.

**Comment by imback82** [comment]
Can you use 2.4.x version of jar?

**Comment by bamurtaugh** [comment]
That fixed it, thank you! I was successful by running:

`spark-submit --class org.apache.spark.deploy.dotnet.DotnetRunner --master local C:\Users\brmurtau\Documents\GitHub\sparkedits\src\scala\microsoft-spark-2.4.x\target\microsoft-spark-2.4.x-0.5.0.jar Microsoft.Spark.CSharp.Examples.exe Sql.Basic %SPARK_HOME%\examples\src\main\resources\people.json`

For updating the doc, should we update the submit command's jar to something like:

`C:\github\dotnet-spark\src\scala\microsoft-spark-2.4.x\target\microsoft-spark-2.4.x-0.5.0.jar`

rather than 

`C:\github\dotnet-spark\src\scala\microsoft-spark-2.3.x\target\microsoft-spark-2.3.x-1.0.0-alpha.jar`?

**Comment by imback82** [comment]
yes, we can update it. `1.0.0-alpha` seems very old. :)
